example/src/config.ts

```diff
@@ -0,0 +1,137 @@
+import type { NavigatorScreenParams } from '@react-navigation/core';
```

Comment:
I'd call this file `screens.tsx`. Also, we use `.tsx` extension for all of the files.